x-pack/plugins/apm/public/components/app/transaction_overview/index.tsx

```diff
@@ -62,15 +72,24 @@ export function TransactionOverview() {
           <EuiSpacer size="s" />
         </>
       )}
-      <TransactionCharts
-        kuery={kuery}
-        environment={environment}
-        start={start}
-        end={end}
-        isServerlessContext={isServerless}
-        comparisonEnabled={comparisonEnabled}
-        offset={offset}
-      />
+      {!isMobileAgent ? (
+        <TransactionCharts
+          kuery={kuery}
+          environment={environment}
+          start={start}
+          end={end}
+          isServerlessContext={isServerless}
+          comparisonEnabled={comparisonEnabled}
+          offset={offset}
+        />
+      ) : (
+        <MobileAgentTransactionCharts
+          kuery={kuery}
+          environment={environment}
+          start={start}
+          end={end}
+        />
+      )}
```

Comment:
Same here =D

Reply:
Updated in https://github.com/elastic/kibana/pull/143697/commits/9cab05301c4352db61467fd0923f573308443f48